src/game/editor/editor.cpp

```diff
@@ -167,41 +167,33 @@ int CLayerGroup::SwapLayers(int Index0, int Index1)
 
 void CEditorImage::AnalyseTileFlags()
 {
-	if(m_Format == CImageInfo::FORMAT_RGB)
-	{
-		for(int i = 0; i < 256; ++i)
-			m_aTileFlags[i] = TILEFLAG_OPAQUE;
-	}
-	else
-	{
-		mem_zero(m_aTileFlags, sizeof(m_aTileFlags));
+	mem_zero(m_aTileFlags, sizeof(m_aTileFlags));
 
-		int tw = m_Width/16; // tilesizes
-		int th = m_Height/16;
-		if(tw == th)
-		{
-			unsigned char *pPixelData = (unsigned char *)m_pData;
+	int tw = m_Width/16; // tilesizes
+	int th = m_Height/16;
+	if(tw == th)
+	{
+		unsigned char *pPixelData = (unsigned char *)m_pData;
 
-			int TileID = 0;
-			for(int ty = 0; ty < 16; ty++)
-				for(int tx = 0; tx < 16; tx++, TileID++)
-				{
-					bool Opaque = true;
-					for(int x = 0; x < tw; x++)
-						for(int y = 0; y < th; y++)
+		int TileID = 0;
+		for(int ty = 0; ty < 16; ty++)
+			for(int tx = 0; tx < 16; tx++, TileID++)
+			{
+				bool Opaque = true;
+				for(int x = 0; x < tw; x++)
+					for(int y = 0; y < th; y++)
+					{
+						int p = (ty*tw+y)*m_Width + tx*tw+x;
+						if(pPixelData[p*4+3] < 250)
 						{
-							int p = (ty*tw+y)*m_Width + tx*tw+x;
-							if(pPixelData[p*4+3] < 250)
-							{
-								Opaque = false;
-								break;
```

Comment:
This break doesn't really do much since it only breaks the inner for-loop

Reply:
(not really related to the point of this pull request but could still be fixed ^^)

Reply:
If removing rgb should be a isolated pull request for whatever reason I'll just add a pull request after this one

Reply:
Should also break the outer loop to avoid unnecessary iterations. For example append `&& Opaque` to both for-conditions and remove the explicit break.